Doppler.HtmlEditorApi/Domain/DopplerHtmlDocument.cs

```diff
@@ -89,6 +89,30 @@ public void SanitizeTrackableLinks()
         }
     }
 
+    /// <summary>
+    /// Fix html code exported by unlayer for a right doppler process to create a abandoned cart automation
+    /// - Remove a repeated items carts
```

Comment:
No estoy seguro, pero me parece que suena un poco mejor:
```suggestion
    /// Sanitize the HTML code exported by Unlayer for the right processing of Doppler.
    /// - Remove repeated cart item tags
```